builds/checkin/dotnet.yaml

```diff
@@ -67,3 +67,44 @@ jobs:
           testRunner: VSTest
           testResultsFiles: '**/TestResults/*.trx'
         condition: succeededOrFailed()
+
+################################################################################
+  - job: code_coverage
+################################################################################
+    displayName: Code coverage
+    variables:
+      coverage.goal: 64
+    pool:
+      vmImage: windows-2019
+    steps:
+      - task: DotNetCoreCLI@2
+        displayName: Build test projects
+        inputs:
+          command: build
+          projects: edge-*\test\Microsoft.Azure.Devices.*Test\Microsoft.Azure.Devices.*Test.csproj
```

Comment:
we also have unit tests in test\modules\Modules.Test\Modules.Test.csproj

Reply:
Oh right, that throws this glob spec off...let me rework it.

Reply:
Fixed. Note that I went from building just test project to building everything because it's actually faster. 😃